packages/synapse-react-client/src/components/DownloadCart/EnableTwoFaRequirementCard.tsx

```diff
@@ -37,8 +40,9 @@ export function EnableTwoFaRequirementCard(
           <WideButton
             variant="contained"
             onClick={() => setIsShowingAccessRequirement(true)}
+            disabled={isEnrolledIn2Fa}
           >
-            Start
+            {isEnrolledIn2Fa ? 'Complete' : 'Start'}
```

Comment:
These action cards may still be shown on the screen after they are completed, so they should themselves indicate when the action has been fulfilled.